test/manual/amp-image-lightbox.amp.html

```diff
@@ -134,6 +130,19 @@ <h1>Image Lightbox</h1>
       on="tap:image-lightbox1"></amp-img>
   </div>
 
+
+  <div>
+    <amp-img
+      src="https://picsum.photos/300/256"
+      layout='fixed-height'
```

Comment:
nit: prefer double quotes for html attributes https://google.github.io/styleguide/htmlcssguide.html#HTML_Quotation_Marks